pkg/driver/controller.go

```diff
@@ -553,20 +553,17 @@ func (cs *controller) ControllerExpandVolume(
 	 * to the specified volume ID is already larger than or equal to the target
 	 * capacity of the expansion request, the plugin should reply 0 OK.
 	 */
-	if volsize >= updatedSize {
-		return csipayload.NewControllerExpandVolumeResponseBuilder().
-			WithCapacityBytes(volsize).
-			Build(), nil
-	}
-
-	if err := lvm.ResizeVolume(vol, updatedSize); err != nil {
-		return nil, status.Errorf(
-			codes.Internal,
-			"failed to handle ControllerExpandVolumeRequest for %s, {%s}",
-			volumeID,
-			err.Error(),
-		)
+	if volsize < updatedSize {
+		if err := lvm.ResizeVolume(vol, updatedSize); err != nil {
+			return nil, status.Errorf(
+		    	codes.Internal,
+		    	"failed to handle ControllerExpandVolumeRequest for %s, {%s}",
+		    	volumeID,
+		    	err.Error(),
+		    )
+		}
```

Comment:
I am not seeing a reason to have this change. Also it removes the bit which makes the call idempotent. I would request you to explain this in the issue.

Reply:
And this condition `volsize < updatedSize` already gets handled by the sidecars, we don't need to do it again.